kineticAR.md

```diff
@@ -0,0 +1,285 @@
+
+If you are using an MIR100 robot, we already have an interface for it. Follow the next steps:
+
+Run de server:
+
+```
+node server.js
+```
+
+Go to the server in the browser:
+
+```
+localhost:8080
+```
+
+
+Select 'Manage Hardware Interfaces'.
+
+You will see a list of the hardware interfaces that are on your server. You can turn them on and off.
+
+Turn the MIR100 hardware interface on and click on the yellow gear for configuration.
+You will see the configuration parameters for the MIR100 robot as follows:
+
+![Image of VST tool](resources/img/mir_config.png) 
+
+Modify the parameters to match your configuration.
+Modify the IP and port so that they match the one from your robot.
+Change the object name to the name you used for your Object.
+Finally, set enableMIRConnection to true, so that the software tries to connect to the robot.
+
+Once this is all setup, stop your server. Go to the next section: Testing.
+
+## Testing
+
+At this point, you have everything setup and you should test to make sure that everything works together.
+Follow these steps to ensure that your code runs properly.
+
+* Make sure the computer running the server, the smartphone with the Reality Editor application, and your robot are on the same network.
+* On your terminal, navigate to the folder 'vuforia-spatial-edge-server/'
+* Run the server by typing:
+
+```
+node server.js
+```
+
+If you are using an MIR100 robot or have any other robot with configuration parameters on the server browser interface, access it in order to make sure the robot has been connected.
+
+* Open the browser and go to:
+```
+localhost:8080
+```
+
+* Go to Manage hardware interfaces
+* Your robot hardware interface should be ON.
+* If you are using the MIR100 hardware interface, click on the yellow gear.
+* Check that the isRobotConnected parameter is set to ON. This means the connection to the robot was successful
+
+* Open the Reality Editor application on the phone
+* On the side menu on the phone, click on the configuration gear
+* Go to 'Found Objects'
+
+Here, you will see a list of Objects that the server is looking for. Your object should be on this list.<br />
+If your object is not on this list, something went wrong. Check out our troubleshooting section.
+
+If your object is on the list, point with the phone at your physical object (aka your robot).
+If your object target works, the main UI of the KineticAR tool should show up on the device.
+
+![Image of VST tool](resources/img/mir.PNG) 
+
+## Authors
+
+* **[Anna Fuste](https://github.com/afustePTC)**
+
+See also the list of [contributors](https://github.com/your/project/contributors) who participated in this project.
+
+## License
+
+This project is licensed under the MPL 2.0 License - see the [LICENSE.md](LICENSE.md) file for details
+
+## Acknowledgments
+
+* Hat tip to anyone whose code was used
+* We thank Rob Stark for being on our wall watching all of us during the development process
```

Comment:
Hahaha I like this acknowledgement

Reply:
I knew you would...
Oh I spelled his name wrong xD I'll fix that